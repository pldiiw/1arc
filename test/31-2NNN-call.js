'use strict';

let engine = require('engine').initialize();
const call = require('instruction').call;

engine.pointer = 4;
engine.pc = 100;

const engine1 = call(engine, 500);

console.log(engine1.pointer);
console.log(engine1.stack[4]);
console.log(engine1.pc);
