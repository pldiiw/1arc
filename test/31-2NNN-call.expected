5
102
500
